nixos/modules/programs/pay-respects.nix

```diff
@@ -41,11 +47,49 @@ in
           The default value is `f`, but you can use anything else as well.
         '';
       };
+      runtimeRules = mkOption {
+        type = listOf type;
+        default = [ ];
+        example = literalExpression ''
+          [
+            {
+              command = "xl";
+              match_err = [
+                {
+                  pattern = [
+                    "libxl: error: .* Permission denied"
```

Comment:
Are you tring to use regular expression here?

RegEx isn't used for general pattern matching, as the underlying regex library is regex_lite, which prioritize binary size and have worse performance. Matching only `Permission denied` in pattern and match `libxl: error:` in suggest condition (`err_contains`) can achieve the same objective with probably less time complexity `~2n` vs `~n^2`

Reply:
Thanks, that was my bad. I'm now suggesting `permission denied` in the pattern and and using `err_contains` to restrict it to `libxl` errors.